Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -1080,6 +1079,33 @@ void ConsoleVariableModel::ClearModifiedRows()
     m_modifiedRows.clear();
 }
 
+AZ::ConsoleCommandInvokedEvent::Handler ConsoleVariableEditor::m_commandInvokedHandler(
+    [](AZStd::string_view command,
+       const AZ::ConsoleCommandContainer&,
+       AZ::ConsoleFunctorFlags,
+       AZ::ConsoleInvokedFrom)
+    {
+        // stop at the first space, if it exists, or npos if it doesn't
+        const auto spacePosition = command.find_first_of(" ");
+
+        AZStd::string variableName = command.substr(0, spacePosition);
```

Comment:
The `command` variable will not have a space in it.
It is passed separately from the command arguments that are passed via the AZ::ConsoleCommandContainer parameter.